typings/json-signatures.d.ts

```diff
@@ -0,0 +1,11 @@
+declare module 'jsonld-signatures' {
+  let suites: any
```

Comment:
I noticed you're `.map`ping on `suites` somewhere. If it's an array of stuff, maybe `ReadonlyArray<any>` would improve type declaration slightly.

Taking a wild guess but is it a `ReadonlyArray<string>`?

Reply:
I believe so

Reply:
It's an object: 
```
const suites = {
  EcdsaKoblitzSignature2016: require('./suites/EcdsaKoblitzSignature2016'),
  Ed25519Signature2018: require('./suites/Ed25519Signature2018'),
  LinkedDataSignature: require('./suites/LinkedDataSignature'),
  LinkedDataSignature2015: require('./suites/LinkedDataSignature2015'),
  GraphSignature2012: require('./suites/GraphSignature2012'),
  RsaSignature2018: require('./suites/RsaSignature2018')
};
```
And I'm using it here: https://github.com/poetapp/poet-js/pull/150/files#diff-3f45022c70dd5ce8f6c044da93286d6cR79

Reply:
So I am setting it to `object`